components/GlobalButton/index.js

```diff
@@ -0,0 +1,9 @@
+import * as Styled from "./GlobalButton.styled";
+
+export default function GlobalButton({ name, onClick, type, textColor }) {
+  return (
+    <Styled.globalButton onClick={onClick} type={type} textColor={textColor}>
```

Comment:
I think renaming the component to just Button is more concise, the global does not really convey a meaning.